PYME/Analysis/gen_sCMOS_maps.py

```diff
@@ -42,12 +49,27 @@ def _meanvards(dataSource, start=0, end=-1):
 
     return (m,v)
 
-def map_filename(mdh, type):
-    if type != 'flatfield':
+def map_filename(mdh, stem):
```

Comment:
This seems like an un-necessary and confusing change in signature. I'm pretty sure that all uses of this code haven't been updated (might not strictly matter, as I suspect the arguments are passed as positional rather than keyword args, but can't be certain).

Reply:
You don't think I checked that? Nowhere this code is used is it used by keyword. No one will be confused unless they have no business looking at it. Overriding `type()` is sloppy, and I didn't think anyone would care if I changed it

Reply:
I find `type` a much more intuitive name than `stem` in this context. If we are really worried about clobbering the builtin `type` (for a small function like this it's unlikely to be an issue) we could potentially use something like `map_type`.